src/app/functions/server/getCountryCode.ts

```diff
@@ -19,9 +19,9 @@ export function getCountryCode(
   const acceptLanguage = headers.get("Accept-Language");
   if (acceptLanguage) {
     const acceptedLocales = acceptLanguage.split(",");
-    const primaryLocale = acceptedLocales[0] ?? "";
+    const primaryLocale = acceptedLocales[0];
     const [locale, _weight] = primaryLocale.split(";");
-    const [_language, country] = (locale ?? "").split("-");
+    const [_language, country] = locale.split("-");
```

Comment:
Q: why these changes?
Seems possibly unrelated to code coverage thresholds.

Reply:
The `?? ""` was marked as uncovered, and didn't actually do anything (`.split` always returns an array with at least one element). So rather than adding an exception, I figured I'd apply this small change to get it to 100% coverage.